recipes/c-client/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    @property
+    def _is_msvc(self):
+        return self._settings_build.compiler in ("Visual Studio", "msvc")
+
+    def validate(self):
+        if self._settings_build.os == "Windows" and not self._is_msvc:
+            raise ConanInvalidConfiguration(
+                "c-client is setup to build only with MSVC on Windows"
+            )
+        # FIXME: need krb5 recipe
+        if self.settings.os == "Macos":
+            raise ConanInvalidConfiguration(
+                "c-client depends on krb5 on MacOS and it's not packaged by "
+                "Conan yet"
+            )
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if not self._is_msvc:
+            self.requires("openssl/1.1.1m")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+
+    def _patch_msvc(self):
+        opt_flags = "/O2 /Ob2 /DNDEBUG"
+        if self.settings.build_type == "Debug":
+            opt_flags = "/Zi /Ob0 /Od /RTC1"
+        runtime = f"/{self.settings.compiler.runtime}"
+        # NOTE: boatloads of warnings for truncation, sign mismatch,
+        #       implicit conversions, just the usual C things
+        warnings = \
+            "/W3 /wd4267 /wd4244 /wd4273 /wd4311 /wd4312 /wd4133 /wd4028"
+        cflags = f"{runtime} {warnings} /GS {opt_flags}"
+        search = "EXTRACFLAGS ="
+        replace = f"EXTRACFLAGS = {cflags}"
+        tools.replace_in_file(r"src\osdep\nt\makefile.w2k", search, replace)
+
+    def _build_msvc(self):
+        make = "nmake /nologo /f makefile.w2k"
+        with tools.vcvars(self):
+            self.run(f"{make} c-client", run_environment=True)
+            self.run(make, cwd="c-client", run_environment=True)
+
+    def _chmod_x(self, path):
+        os.chmod(path, os.stat(path).st_mode | stat.S_IEXEC)
+
+    def _touch(self, path):
+        with open(path, "a", encoding=None): pass
+
+    def _build_unix(self):
+        self._touch("ip6")
+        self._chmod_x("tools/an")
+        self._chmod_x("tools/ua")
+        unix = "src/osdep/unix"
+        self._chmod_x(f"{unix}/drivers")
+        self._chmod_x(f"{unix}/mkauths")
+        search = "SSLDIR=/usr/local/ssl"
+        ssldir = self.deps_cpp_info["openssl"].rootpath
+        tools.replace_in_file(f"{unix}/Makefile", search, f"SSLDIR={ssldir}")
+        # This is from the Homebrew Formula
+        tools.replace_in_file(
+            "src/osdep/unix/ssl_unix.c",
+            "#include <x509v3.h>\n#include <ssl.h>",
+            "#include <ssl.h>\n#include <x509v3.h>"
+        )
+        target = "osx" if self.settings.os == "Macos" else "slx"
+        args = ["IP=6", "-j1"]
```

Comment:
```suggestion
        args = ["IP=6", "-j{}".format(tools.cpu_count())]
```

file:///home/uilian/Development/conan/docs/_build/html/reference/tools.html#tools-cpu-count

Reply:
The build is set to one job, because there were problems in parallel builds. Maybe I should have left a `NOTE:` above.

Reply:
Yes, it would be nice.